gemviz/bluesky_runs_catalog.py

```diff
@@ -0,0 +1,119 @@
+"""
+MVC implementation of CatalogOfBlueskyRuns.
+
+* BRC: BlueskyRunsCatalog
+* MVC: Model View Controller
+
+.. autosummary::
+
+    ~BRC_MVC
+"""
+
+import time
+from functools import partial
+
+from PyQt5 import QtWidgets
+
+import utils
+
+
+class BRC_MVC(QtWidgets.QWidget):
+    """MVC class for CatalogOfBlueskyRuns."""
+
+    # UI file name matches this module, different extension
+    ui_file = utils.getUiFileName(__file__)
+    motion_wait_time = 1  # wait for splitter motion to stop to update settings
+
+    def __init__(self, parent):
+        self.parent = parent
+
+        super().__init__()
+        utils.myLoadUi(self.ui_file, baseinstance=self)
+        self.setup()
+
+    def setup(self):
+        from app_settings import settings
+        from gemviz.bluesky_runs_catalog_run_viz import BRCRunVisualization
+        from gemviz.bluesky_runs_catalog_search import BRCSearchPanel
+        from gemviz.bluesky_runs_catalog_table_view import BRCTableView
+
+        self.brc_search_panel = BRCSearchPanel(self)
+        layout = self.filter_groupbox.layout()
+        layout.addWidget(self.brc_search_panel)
+        self.brc_search_panel.catalogSelected(self.catalog().item["id"])
```

Comment:
From what I understand,  `self.catalog()` return self.parent.catalog() which is basically `mainwindow.catalog()`, is that correct? 
But what is `.item["id"]` here? What it is doing?

Reply:
Might be the name of the catalog, as reported by tiled.  Is the name available by an easier approach?  

`self._catalogSelected` is defined.  Is this available yet?  Also, this attribute name could be changed to `self._catalogName` and then provide accessor method `self.catalogName()`.